app/assets/stylesheets/editor/math.scss

```diff
@@ -0,0 +1,42 @@
+.post-editor {
```

Comment:
we shouldnt be doing this. can this not be a high level thing?

Reply:
> we shouldnt be doing this

can you clarify what you mean by "doing this"? did you mean to say `.post-editor` styles shouldn't live in `math.scss`? would you prefer all `.post-editor` styles live in `posts.scss`?

`counter-reset: math-block` needs to be set on `.post-editor` because it is used to count and number the math_blocks in the post